packages/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -1944,10 +1944,27 @@ export default class Meeting extends StatelessWebexPlugin {
             this.inMeetingActions.get()
           );
         }
+
+        this.handleDataChannelUrlChange(payload.info.datachannelUrl);
```

Comment:
would recommend to move this logic to join as we only need llm when meeting is actively joined meeting

Reply:
This needs to be here to disconnect/reconnect the data channel when we move in and out of breakout sessions. However I'll put a check in ensureLLMConnected to only connect if we are joined